index.src.html

```diff
@@ -260,7 +260,7 @@ <h3 dfn export id="referrer-policy-no-referrer" oldids="referrer-policy-state-no
 
   <h3 dfn export id="referrer-policy-no-referrer-when-downgrade" oldids="referrer-policy-state-no-referrer-when-downgrade">"<code>no-referrer-when-downgrade</code>"</h3>
 
-  The <a>"<code>no-referrer-when-downgrade</code>"</a> policy sends a full URL
+  The <a>"<code>no-referrer-when-downgrade</code>"</a> policy sends a full URL, <a href="#strip-url">stripped for use as a referrer</a>, 
```

Comment:
Unfortunately this document doesn't really enforce a strict line-number wrapping, but I'd like to try and keep to 100 characters whenever changes are made - could you please wrap this?

Reply:
Done